_topic_map.yml

```diff
@@ -1685,8 +1685,8 @@ Topics:
     File: cluster-logging-curator
   - Name: Maintenance and support
     File: cluster-logging-maintenance-support
-- Name: Viewing cluster logs
-  File: cluster-logging-viewing
+- Name: Viewing logs for a specific resource
+  File: resource-logs-viewing
```

Comment:
Just caught this while looking over additional commends. For the new file name, do you want to keep it prepended with "cluster-logging" because it is still in the cluster logging folder? For example, "cluster-logging-viewing-resource-logs"?

Reply:
@lbarbeevargas I was concerned that might give the wrong impression that the feature is part of cluster logging, when it's not. Does that break any rules/conventions?

Reply:
I'm not seeing any specific guidance other than the link you referenced and [Assembly/module file names](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assemblymodule-file-names). I would say that if this is a procedure assembly/module, then you could change them to something like `viewing-resource-logs` and `viewing-resource-logs-cli-console`.